rpc/transaction.go

```diff
@@ -791,8 +793,6 @@ func adaptTransactionStatus(txStatus *starknet.TransactionStatus) (*TransactionS
 		status.Execution = TxnSuccess
 	case starknet.Reverted:
 		status.Execution = TxnFailure
-	case starknet.Rejected:
-		status.Finality = TxnStatusRejected
```

Comment:
are you sure that `Rejected` is part of finality status? according to this image it should be in `ExecutionStatus`:
![image](https://github.com/user-attachments/assets/6773fbab-6c7e-479c-ab02-d874bd2ae6e4)

Reply:
https://docs.starknet.io/architecture-and-concepts/network-architecture/transaction-life-cycle/

Reply:
This doc suggests it should be apart of Finality Status

Reply:
Same with the spec https://github.com/starkware-libs/starknet-specs/blob/v0.7.1/api/starknet_api_openrpc.json#L3200-L3210